packages/timere/timere.0.2.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "OCaml date time handling and reasoning library"
+description: """
+Features:
+
+- Timestamp and date time handling with platform independent time zone support
+
+  - Subset of the IANA time zone database is built into this library
+
+- Reasoning over time intervals via `timere` objects/expressions, examples:
+
+  - Pattern matching time and intervals. These work across DST boundaries.
+ 
+  - Intersection and union
+
+  - Chunking at year or month boundary, or in fixed sizes
+
+  - Evaluate (sub)expressions with a different time zone (e.g. intersection of 9am to 5pm of Sydney and 9am to 5pm of New York)
+"""
+maintainer: "Darren Ldl <darrenldldev@gmail.com>"
+authors: "Daypack developers"
+license: "MIT"
+homepage: "https://github.com/daypack-dev/timere"
+bug-reports: "https://github.com/daypack-dev/timere/issues"
+dev-repo: "git+https://github.com/daypack-dev/timere"
+depends: [
+  "ocaml" {>= "4.06.1"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
```
Module `Int` was introduced in 4.08

Reply:
Are you ok with this, or you want to update the code to support 4.06?

Reply:
I'll fix the code - forgot to test the build on 4.06.1